cfme/tests/containers/test_relationships_tables.py

```diff
@@ -180,8 +167,7 @@ def test_images_rel(provider, rel):
         obj = Image(name, provider)
 
         val = obj.get_detail('Relationships', rel)
-        if val == '0' or val == 'Unknown image source':
-            continue
+        assert val != 'Unknown image source'
```

Comment:
So there is https://bugzilla.redhat.com/show_bug.cgi?id=1365878 currently in `MODIFIED` state... which means this assert will always fail.
Why would you want a test that will always fail? What you want to do is add a `pytest.mark.meta(blockers=` marker, see here: https://github.com/ManageIQ/integration_tests/blob/3f826f7f3bac9a9effc0494e61c365ba6d1c0606/cfme/tests/infrastructure/test_providers.py#L106 - this will make the test skip until the BZ is resolved.